test/framework/src/main/java/org/elasticsearch/common/logging/JsonLogLine.java

```diff
@@ -30,7 +30,7 @@
  * Parsing log lines with this class confirms the json format of logs
  */
 public class JsonLogLine {
-    public static final ObjectParser<JsonLogLine, Void> PARSER = createParser(false);
+    public static final ObjectParser<JsonLogLine, Void> PARSER = createParser(true);
```

Comment:
Why do we need this leniency? Shoudln't our custom format mean all the fields are known?

Reply:
some of the fields are defined in a properties file. 
Like in deprecation logs - a log message can have x-opaque-id or slow logs can have custom fields there too.
Initially I started with just server logs where I only expected 10fields. But now I also use this for Deprecation logs and slow logs so I made it more flexible.
This class is for testing only, but there is another question worth considering. Should we limit and validate fields defined in `log4j2.properties`? Like we wouldn't allow people to randomly modify fields for instance in this property:
```
appender.index_indexing_slowlog_rolling.layout.esmessagefields=message,took,took_millis,doc_type,id,routing,source
```